server.js

```diff
@@ -551,10 +551,16 @@ camp.route(/^\/sonar\/(http|https)\/(.*)\/(.*)\/(.*)\.(svg|png|gif|jpg|json)$/,
       var buildType = match[3];  // eg, `ru.yandex.qatools.allure:allure-core:master`.
       var metric = match[4];
       var format = match[5];
-      var realMetric = metric === 'tech_debt' ? 'sqale_debt_ratio' : metric
+      var sonarMetric = metric;
+      if (metric === 'tech_debt') {
+        sonarMetric = 'sqale_debt_ratio';
+      } else if (metric === 'lines') {
+        sonarMetric = 'ncloc';
```

Comment:
Let's settle on 'lines' keeping SonarQube's meaning, and 'loc' (and 'ncloc') doing what SonarQube calls ncloc. We can call it 'loc' for both, as well.

What do you think?